### DISCUSSION TITLE
Trigger button for GitHub workflow missing for Workflow dispatch

### DISCUSSION THREAD

**Discussion by abhinavan1812** [community/community#35399] [Actions]
Hello @github-staff
I am using workflow dispatch as defined in GitHub docs for workflow triggers.
But in actions tab I am not able to see trigger button for the same.

I want to trigger the workflow from every branch manually whenever needed by a developer

**Comment by DSDmark** [comment]
Once you create a workflow, you can trigger workflow by manually 


![Screenshot from 2022-10-07 13-32-45](https://user-images.githubusercontent.com/85744816/194504201-c1f7d13d-ac76-4759-babd-c81b53a73802.png)
